packages/app/src/pages/PoolPage/PoolCurrentPosition.tsx

```diff
@@ -0,0 +1,38 @@
+import { PreviewTable, PreviewItem } from "~/components/PreviewTable";
+import { TokenIcon } from "~/components/TokenIcon";
+import { useCoinMetadata } from "~/hooks/useCoinMetadata";
+import { useUserPositions } from "~/hooks/useUserPositions";
+
+export const PoolCurrentPosition = () => {
+  const info = useUserPositions();
+  const { coinMetaData } = useCoinMetadata({ symbol: "ETH/DAI" });
```

Comment:
We have a const for this

Reply:
could you tell where is it? I searched but only see other places using raw `ETH/DAI` but not using any default const param

Reply:
on file CoinsMeta